autopush/router/webpush.py

```diff
@@ -53,8 +53,9 @@ def _crypto_headers(self, notification):
         # they're present to avoid empty strings.
         for name in ["encryption-key", "crypto-key"]:
             if name in headers:
-                # Normalize hyphens in header names.
-                data[name.replace("-", "_")] = headers[name]
+                # NOTE: The client code expects all header keys to be lower
+                # case and s/-/_/.
+                data[name.lower().replace("-", "_")] = headers[name]
```

Comment:
This just makes it more explicit, right? It looks like the names are already lowercase in the list on line 54.